event_files/prediction_metrics.json

```diff
@@ -0,0 +1,212 @@
+[
+    {
+        "MetricName": "PRD_RETIRED.CONTROL_FLOW.PKI",
+        "BriefDescription": "The rate of control flow instructions retired per kilo instructions",
+        "MetricExpr": "PRD_RETIRED.CONTROL_FLOW / RETIRED.INST * 1000",
+        "MetricGroup": "Prediction",
+        "ScaleUnit": "1PKI"
+    },
+    {
+        "MetricName": "PRD_RETIRED.CONTROL_FLOW.MPKI",
+        "BriefDescription": "The rate of control flow instructions mis-predicted per kilo instructions",
+        "MetricExpr": "PRD_RETIRED.CONTROL_FLOW.MISPRED / RETIRED.INST * 1000",
+        "MetricGroup": "Prediction",
+        "ScaleUnit": "1MPKI"
+    },
+    {
+        "MetricName": "PRD_RETIRED.CONTROL_FLOW.MISPRED_RATE",
+        "BriefDescription": "The rate of control flow instructions mis-predicted to the overall control flow instructions",
+        "MetricExpr": "PRD_RETIRED.CONTROL_FLOW.MISPRED / PRD_RETIRED.CONTROL_FLOW.RETIRED",
```

Comment:
Mispred rate should be based on the percentage of *predicted* branches/jumps that are mispredicted, right?  So we'd want to exclude direct branches in the denominator.